builds/e2e/templates/nested-get-device-name.yaml

```diff
@@ -4,7 +4,7 @@ steps:
     inputs:
       targetType: inline
       script: |
-        device_name=$(az vm show -d  -g "iotedge-deploy" -n $(hostname) --query fqdns)
+        device_name=$(ip route get 8.8.8.8 | sed -n '/src/{s/.*src *\([^ ]*\).*/\1/p;q}')
```

Comment:
what is the change here?

Reply:
Getting the public IP address instead of using the FQDN (which is empty string for this VM).